parmoo/tests/unit_tests/test_moop_grads.py

```diff
@@ -189,11 +193,11 @@ def dc2(x, s):
     # at the interpolation nodes...
     x = moop1._embed({'x1': 1, 'x2': 1, 'x3': 1})
     xx = moop2._embed({'x1': 1, 'x2': 1, 'x3': 1})
-    assert (np.linalg.norm(eval_pen_jac(moop1, x) - eval_pen_jac(moop2, xx) < 1.0e-8))
+    assert (np.linalg.norm(eval_pen_jac(moop1, x) -
+                           eval_pen_jac(moop2, xx) * 2) < 1.e-8)
```

Comment:
Change this 1.e-8 to something more type-/precision-adaptable?
e.g., 
eps = jnp.finfo(jnp.zeros(1).dtype).eps

Reply:
Yes, I agree -- it is actually a problem as we've start working in jax, since jax default precision is single so this tolerance is too low.  There are a lot of places where this occurs, and I'm not sure I have time to work on that in the immediate future.  What do you think of creating a new issue for now?

Reply:
Issue sounds good, pointing to [parmoo/tests/unit_tests/test_moop_grads.py](https://github.com/parmoo/parmoo/pull/87/files/8d6667819bee9cccede468508a7d335ac0b43b30#diff-43061fcfc6ccc31f31a28c263ea82324351992420f0b049edb8a71d472364497) as a specific example